source/uwp/Renderer/lib/ActionHelpers.h

```diff
@@ -4,79 +4,104 @@
 
 namespace AdaptiveCards::Rendering::Uwp::ActionHelpers
 {
-    HRESULT BuildAction(_In_ ABI::AdaptiveCards::ObjectModel::Uwp::IAdaptiveActionElement* adaptiveActionElement,
-                        _In_ ABI::AdaptiveCards::Rendering::Uwp::IAdaptiveRenderContext* renderContext,
-                        _In_ ABI::AdaptiveCards::Rendering::Uwp::IAdaptiveRenderArgs* renderArgs,
-                        bool isOverflowActionButton,
-                        _Outptr_ ABI::Windows::UI::Xaml::IUIElement** actionControl);
-
-    HRESULT GetButtonMargin(_In_ ABI::AdaptiveCards::Rendering::Uwp::IAdaptiveActionsConfig* actionsConfig,
-                            ABI::Windows::UI::Xaml::Thickness& buttonMargin) noexcept;
-
-    void ArrangeButtonContent(_In_ ABI::AdaptiveCards::ObjectModel::Uwp::IAdaptiveActionElement* action,
-                              _In_ ABI::AdaptiveCards::Rendering::Uwp::IAdaptiveActionsConfig* actionsConfig,
-                              _In_ ABI::AdaptiveCards::Rendering::Uwp::IAdaptiveRenderContext* renderContext,
-                              ABI::AdaptiveCards::ObjectModel::Uwp::ContainerStyle containerStyle,
-                              _In_ ABI::AdaptiveCards::Rendering::Uwp::IAdaptiveHostConfig* hostConfig,
+    winrt::Windows::UI::Xaml::UIElement BuildAction(winrt::AdaptiveCards::ObjectModel::Uwp::IAdaptiveActionElement const& adaptiveActionElement,
+                                                    winrt::AdaptiveCards::Rendering::Uwp::AdaptiveRenderContext const& renderContext,
```

Comment:
style nits, indent 4, instead of aligning with the `(`

Reply:
@beervoley - we may need to update our `.clang-format` - sync with me on how to do that if you need help...